modules/claims_api/lib/custom_error.rb

```diff
@@ -29,18 +29,9 @@ def build_error
 
     private
 
-    def raise_backend_exception(key = 'EVSS')
+    def raise_backend_exception(_key = 'EVSS')
```

Comment:
@stiehlrod  If `key` isn't used anymore can we just take it out?

Reply:
It appears that the method `raise_backend_exception` does not need any arguments sent to it so the lines calling this above can probably be cleaned up to not have any argument.  So lines 17,21 and 24 and then this `_key` parameter that is not used can also probably just be cleaned out to remove any confusion on what it is doing there.

Reply:
we also carried this 'EVSS' key over from the past error messages _when we used EVSS._ Since we no longer technically use EVSS, this could probably be modified. For now we can continue this process but worth thinking about.

Reply:
Ya, it is just that `key` variable is not used anywhere in this method anymore so no point in having it in there as a default, also no use in passing anything to this method because it doesn't utilize any parameters anymore.